packages/sync-repo-settings/src/bin/commands/local-sync.ts

```diff
@@ -0,0 +1,67 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import yargs = require('yargs');
```

Comment:
I am not a huge fan of the nested directory structure here.  In `policy` there was a single `cli.ts` that just encapsulated what the CLI needed to do.

Reply:
Ok, I can collapse into a single file to encapsulate it.